binaries/cli/src/template/cxx/mod.rs

```diff
@@ -51,6 +62,27 @@ fn create_dataflow(name: String, path: Option<PathBuf>) -> Result<(), eyre::ErrR
     Ok(())
 }
 
+fn create_cmakefile(root: PathBuf, use_path_deps: bool) -> Result<(), eyre::ErrReport> {
+    const CMAKEFILE: &str = include_str!("cmake-template.txt");
+
+    let cmake_file = if use_path_deps {
+        let manifest_dir = Path::new(env!("CARGO_MANIFEST_DIR"));
+        let workspace_dir = manifest_dir.parent().unwrap().parent().unwrap();
```

Comment:
Although this is probably never going to fail, could we use eyre.context()
```suggestion
        let workspace_dir = manifest_dir.parent().context("Could not get manifest parent folder")?.parent().context("Could not get manifest grandparent folder")?;
```